anaconda.spec.in

```diff
@@ -41,7 +41,7 @@ Source0: https://github.com/rhinstaller/%{name}/releases/download/%{name}-%{vers
 %define nmver 1.0
 %define pykickstartver 3.47-1
 %define pypartedver 2.5-2
-%define pythonblivetver 1:3.7.1-4
+%define pythonblivetver 1:3.8.1-1
```

Comment:
We need to wait for this build to make it to rawhide (I did a new blivet release today).